server.go

```diff
@@ -312,14 +312,23 @@ func (c *Client) ShutdownServer(id string) error {
 		method: http.MethodPatch,
 		body:   new(map[string]string),
 	}
+
 	err = r.execute(*c, nil)
 	if err != nil {
+		if requestError, ok := err.(*RequestError); ok {
```

Comment:
`err.(*RequestError)` I think this should be `err.(RequestError)` as the `execute` function returns error instance instead of a pointer.

Reply:
I have opened another PR for fixing pointer stuff. You can take a look at it and see what I meant.

Reply:
Thanks, I did the change based on develop branch, so if I use `err.(RequestError)`,  the tests will fail. I'll rebase after merging your PR https://github.com/gridscale/gsclient-go/pull/59.

Reply:
👍